service/src/test/java/bio/terra/externalcreds/services/OAuth2ServiceTest.java

```diff
@@ -60,6 +60,9 @@ void test() {
         oAuth2Service.authorizeWithRefreshToken(
             providerClient, oAuth2AccessTokenResponse.getRefreshToken());
 
+    System.out.println(
+        "refresh token:__________" + tokenResponse.getRefreshToken().getTokenValue());
+
```

Comment:
testing line left in here

Reply:
I thought this was fine

Reply:
ok! If you think it's fine then I'm fine to leave it